src/SFA.DAS.FindEpao.Api.UnitTests/Controllers/Courses/WhenGettingCourseList.cs

```diff
@@ -35,7 +36,10 @@ public class WhenGettingCourseList
             controllerResult.StatusCode.Should().Be((int)HttpStatusCode.OK);
             var model = controllerResult.Value as GetCourseListResponse;
             Assert.IsNotNull(model);
-            model.Courses.Should().BeEquivalentTo(mediatorResult.Courses);
+            model.Courses.Should().BeEquivalentTo(mediatorResult.Courses, options => options
+            .Excluding(c => c.StandardUId)
+            .Excluding(c => c.LarsCode));
+            model.Courses.All(c => c.Id > 0).Should().BeTrue();
```

Comment:
if there's already a test covering the casting of types, just assert that the casts are equivalent, then you don't have to worry about all Id > 0 (they could all be 1 and pass this). e.g. 
`model.Courses.Should().BeEquivalentTo(mediatorResult.Courses.Select(course => (TypeCastTo)course)`

Reply:
There is no equivalent property `Id` in the other object. It maps to `LarsCode` instead of `Id`. Hence we are just making sure here that  the `Id` in outer object is populated.

Reply:
Fixed